distmv/normal.go

```diff
@@ -246,6 +246,29 @@ func (n *Normal) Prob(x []float64) float64 {
 	return math.Exp(n.LogProb(x))
 }
 
+// Quantile returns the multi-dimensional inverse cumulative distribution function.
+// len(x) must equal len(p), and if x is non-nil, len(x) must also equal len(p).
```

Comment:
I still don't understand "len(x) must equal len(p), and if x is non-nil, len(x) must also equal len(p)." the first "len(x)" is presumably supposed to be something else, no?